spec/coder_spec.lua

```diff
@@ -1964,7 +1964,185 @@ describe("Titan code generator", function()
                 assert(0 == test.f(0xdead, math.mininteger))
             ]])
         end)
-end)
+    end)
+
+    describe("#maps", function()
+
+        it("Lua can break a map", function ()
```

Comment:
I forgot to change the name of this test on the commit where I fixed it, sorry!

Reply:
this was actually the name I intended :laughing: I guess a more proper name would be "adding an element of the wrong type in Lua can trigger a runtime error later in Titan"